src/Compilers/CSharp/Portable/Symbols/Source/ImplicitNamedTypeSymbol.cs

```diff
@@ -116,7 +116,7 @@ public override ImmutableArray<CustomModifier> GetTypeArgumentCustomModifiers(in
 
         public sealed override bool AreLocalsZeroed
         {
-            get { return true; }
+            get { return ContainingModule.AreLocalsZeroed; }
```

Comment:
>ContainingModule.AreLocalsZeroed [](start = 25, length = 32)

Unless we can change the input for the constructor (see the comment above), we probably should be prepared to delegate to the containing type, if this type is nested.

Reply:
It makes sense. I updated it to delegate to the ContainingType then, if none is found, to the module.